### PR TITLE
HW6 extra challenge

### PR DESCRIPTION
**Pull request by RoelofWobben**
Some changes to see what is expected of me

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ static async Task Main(string[] args)
         {
             // Put two // in front of the await to comment out Part1And2 once you solved it.
             // Make sure to note down the secret passcode before. You will need it!
-            await Part1And2();
+            //await Part1And2();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927992)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927992&bid=16302843)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927992&bid=16302843)

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927992)

File: Src/BootCamp.Chapter/Algorithm.cs [hunk 2]
```diff
@@ -12,8 +12,8 @@ internal class Algorithm
         /// </summary>
         internal static void OptionalTestCases()
         {
-            // int swapCount = 0;
-            // Sort(new int[0], ref swapCount);
+            int swapCount = 3;
+            Sort(new int[] { 1, 3, 2 } , ref swapCount);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Algorithm.cs]: Codacy Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927992)

**Comment by RoelofWobben** [comment]
I close this one. Additonal challenges are at the moment not for me. Maybe I would look later at it when I have more knowlegde and experience

**Pull request closed without merging**